### PR TITLE
[Docs] iphoneSetFileDataProtection corrections

### PR DESCRIPTION
**Pull request by seaniepie**
- Minor adjustments to spelling and format
- Added description on how to check a files protection level

### CODE REVIEW

File: docs/dictionary/command/iphoneSetFileDataProtection.lcdoc [hunk 1]
```diff
@@ -21,26 +21,27 @@ iphoneSetFileDataProtection specialFolderPath("documents") & "/appointments.txt"
...
--   "complete unless open": The file is fully protected when the device
-    is locked, unless it was already open
--   "complete until first user authentication": The file is fully
-    protected until the user unlocks the device for this first time.
+-   "complete": The file is not accessible, for read or write, while the 
+device is locked.
+-   "complete unless open": The file is fully protected when the device 
+is locked, unless it was already open
+-   "complete until first user authentication": The file is fully 
+protected until the user unlocks the device for this first time.
```
Comment (livecodeali) [inline docs/dictionary/command/iphoneSetFileDataProtection.lcdoc]: These enum descriptions will have to remain unwrapped until we fix http://quality.livecode.com/show_bug.cgi?id=18657

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 5f4fdd3

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 5f4fdd39b76ac99a4368c444e0ee5d822500e266

**Comment by livecode-vulcan** [comment]
😎 test success 5f4fdd39b76ac99a4368c444e0ee5d822500e266
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/483)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/488)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/483)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/483)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/489)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/483)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/483)

**Pull request merged**